ui/app/components/mount-backend-form.js

```diff
@@ -112,18 +112,21 @@ export default Component.extend({
           throw err;
         }
       }
-
+      // TODO CMB: this flash message seems to be an over-generalization, so added the 'if...path !==' check, but want to revisit
+      // TODO should we check instead if config capabilities.capabilites.includes('deny')
       if (!capabilities.get('canUpdate')) {
-        // if there is no sys/mount issue then error is config endpoint.
-        this.flashMessages.warning(
-          'You do not have access to the config endpoint. The secret engine was mounted, but the configuration settings were not saved.'
-        );
```

Comment:
this flash message was displaying when an API error was being returned (from either the path already existing or due to a permissions issue)